nuancier/default_config.py

```diff
@@ -30,6 +30,11 @@
 # Set the time after which the session expires
 PERMANENT_SESSION_LIFETIME = timedelta(hours=1)
 
+# The length of time a CSRF token is valid. Setting this to `None` binds
+# it to the session lifetime, but this only works if you have flask-wtf-0.10.1
```

Comment:
If I recall correctly (and I'm not sure that I do), the setting ``WTF_CSRF_TIME_LIMIT`` doesn't even exist in flask-wtf-0.8 which is what we have in EPEL7. It's documented to exist in 0.10.0 (maybe earlier), but doesn't actually work until 0.10.1 (regardless of whether you set it to ``None`` or some integer). We expose the setting here and if the version of flask-wtf is greater than 0.10.1 we don't touch anything. If it's prior to that version, we use it to set the ``TIME_LIMIT`` on the WTForm directly (which is what flask-wtf eventually does).

If you set it to the same value you set ``PERMANENT_SESSION_LIFETIME`` it's about the same as binding it to the session lifetime.